### PR TITLE
RDKBACCL-291: Webconfig support

### PR DESCRIPTION
**Pull request by hkumar343**
Reason for change: To integrate webconfig feature support in BPIR4.
Test Procedure: Build and test webconfig feature test cases
Risks: None

### CODE REVIEW

File: meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/parodus_read_file.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
```
Comment (cpokuru) [inline meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/parodus_read_file.sh:1]: can this be common file  under meta-cmf-broadband

File: meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/parodus_create_file.sh [hunk 2]
```diff
@@ -0,0 +1,181 @@
+#!/bin/sh
+##########################################################################
```
Comment (cpokuru) [inline meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/parodus_create_file.sh:2]: can this be common under meta-cmf-broadband

File: meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/parodus_read_file.sh (hunk 1 above)
Comment (hkumar343) [reply meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/parodus_read_file.sh:1]: I think yes we can make it common file under meta-cmf-broadband, in that case simultaneously  we need to remove same changes from meta-cmf-raspberrypi.

**Review by cpokuru — APPROVED** [review]

**Comment by cpokuru** [comment]
webconfig changes

File: meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/parodus_read_file.sh (hunk 1 above)
Comment (cpokuru) [reply meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/parodus_read_file.sh:1]: ok

File: meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/parodus_create_file.sh (hunk 2 above)
Comment (cpokuru) [reply meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/parodus2ccsp/parodus_create_file.sh:2]: we will take this as part of cleanup

**Pull request merged**